docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -639,7 +653,7 @@ In this example, the value of the last request path segment is a tenant ID, but
 [[default-tenant-resolver]]
 === Default resolution
 
-The default resolution for a tenant identifier is convention based, whereby the authentication request must include the tenant identifier in the last segment of the request path.
+The default resolution for a tenant identifier is convention-based, whereby the authentication request must include the tenant identifier in the last segment of the request path.
```

Comment:
As agreed with Sheila, it works without `-` for similar forms too

Reply:
Updated. Thanks.